sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -852,7 +852,21 @@ void gnrc_ipv6_netif_init_by_dev(void)
             _add_addr_to_entry(ipv6_if, &addr, 64, 0);
 
         }
-
+#ifdef GNRC_IPV6_STATIC_LLADDR
+        /* parse addr from string and explicitely set a link lokal prefix
+         * if ifnum > 1 each interface will get its own link local address
+         * with GNRC_IPV6_STATIC_LLADDR + i
+         */
+        #define STR(x) #x
+        #define DEFINE2STR(s) STR(s)
```

Comment:
well, you had to define the IP address in the Makefile with escaped quotation marks, which is ugly, too.